tests/music_tests.ts

```diff
@@ -0,0 +1,311 @@
+/**
+ * VexFlow - Music API Tests
+ * Copyright Mohit Muthanna 2010 <mohit@muthanna.com>
+ */
+
+declare const QUnit: any;
+declare const ok: any;
+declare const test: any;
+declare const expect: any;
+declare const equal: any;
+declare const VF: any;
+
+import { Music } from '../src/music';
+
+export const MusicTests = {
+  Start: function () {
+    QUnit.module('Music');
+    test('Valid Notes', this.validNotes);
+    test('Valid Keys', this.validKeys);
+    test('Note Values', this.noteValue);
+    test('Interval Values', this.intervalValue);
+    test('Relative Notes', this.relativeNotes);
+    test('Relative Note Names', this.relativeNoteNames);
+    test('Canonical Notes', this.canonicalNotes);
+    test('Canonical Intervals', this.canonicalNotes);
+    test('Scale Tones', this.scaleTones);
+    test('Scale Intervals', this.scaleIntervals);
```

Comment:
giving it a second thought, I believe that `this` should be replaced by `MusicTests` because this is not the instantiation of a class, would you agree?

Reply:
`this` references to the MusicTests and is same. Why do you think it should be replaced?

Reply:
because I have already migrated `tuning_tests.ts` and I  have seen that with `this` it does not work when you try to call `checkStandard` and `checkStandardBanjo` :)

I decided to make today a second step including `Fraction` and `Tuning` to prove the general concept.

Reply:
It is because you have to bind this to standard function. That is how js works.